packages/react-i18n/src/i18n.ts

```diff
@@ -204,11 +204,15 @@ export class I18n {
   }
 
   translationKeyExists(id: string) {
+    const errorHandler = this.onError;
     try {
+      (this.onError as any) = defaultOnError;
```

Comment:
I know this works but it feels a little icky to me. I'd rather do a proper check for whether the key exists rather than relying on the exception for control flow, especially now that it requires weird mutations of self

Reply:
Tried again, lmk what you think. 😄